### ISSUE TITLE
add player inventory

### ISSUE THREAD

**Issue by saltytext** [saltytext/wg#9]
I guess you can keep the wood that you have chopped up and split your back getting... i guess.

will need to check to see if I can reuse the current create/store/load/delete/print/reset player functions in common fn file.

should be able to add items, remove items, check items.
also put a limit on how much player can hold initially. example, 50 total items but could be 50 different items or 50 of the same item, etc etc.
in other words...no "stacking" of items yet. (storage in some way should be added in the future).

**Comment by saltytext** [comment]
initial functions added for inventory/storage system in v0.0.12

***once this is complete the test page itemtesting.html can be removed.***

**Comment by saltytext** [comment]
viewing inventory added in v0.0.16, accessed by "resting" while exploring.

**Comment by saltytext** [comment]
Remaining function to create is remove items. This will be done in 21.
